darwin/exporter/formats/nifti.py

```diff
@@ -137,50 +137,28 @@ def check_for_error_and_return_imageid(
     image_id : str
 
     """
-    # check if all item slots have the correct file-extension
+    # Check if all item slots have the correct file-extension
     for slot in video_annotation.slots:
         for source_file in slot.source_files:
             filename = Path(source_file["file_name"])
-
-            try:
-                suffixes = filename.suffixes[-2:]
-            except IndexError:
-                suffixes = filename.suffixes
-            if len(suffixes) == 2:
-                if suffixes[0] == ".nii" and suffixes[1] == ".gz":
-                    image_id = str(filename).rstrip("".join(suffixes))
-                else:
-                    return create_error_message_json(
-                        "Two suffixes found but not ending in .nii.gz",
-                        output_dir,
-                        str(filename),
-                    )
-            elif len(suffixes) == 1:
-                if suffixes[0] == ".nii" or suffixes[0] == ".dcm":
-                    image_id = filename.stem
-                else:
-                    return create_error_message_json(
-                        "Misconfigured filename, not ending in .nii or .dcm. Are you sure this is medical data?",
-                        output_dir,
-                        str(filename),
-                    )
-            else:
+            if not (
+                filename.name.endswith(".nii.gz")
+                or filename.name.endswith(".nii")
+                or filename.name.endswith(".dcm")
+            ):
                 return create_error_message_json(
-                    "You are trying to export to nifti. Filename should contain either .nii, .nii.gz or .dcm extension."
-                    "Are you sure this is medical data?",
+                    "Misconfigured filename, not ending in .nii, .nii.gz or .dcm. Are you sure this is medical data?",
                     output_dir,
                     str(filename),
                 )
 
     filename = Path(video_annotation.filename)
-    try:
-        suffixes = filename.suffixes[-2:]
-    except IndexError:
-        suffixes = filename.suffixes
-    if len(suffixes) == 2:
-        image_id = str(filename).rstrip("".join(suffixes))
-    elif len(suffixes) == 1:
-        image_id = str(filename.stem)
```

Comment:
I'm a little unsure about this. Currently, if `.nii.gz` then we include this in the `image_id`

Otherwise, if `.dcm` or `.nii`, we don't include this in the `image_id`. Why is this?

Reply:
> Currently, if .nii.gz then we include this in the image_id

Is that actually true? `image_id = str(filename).rstrip("".join(suffixes))` would strip out `.nii.gz` 🤔

Reply:
Actually yes that is correct

Just tested and the behaviour now is:
- If any slot has any ending other than `.nii`, `.dcm`, or `.nii.gz`, the exporter fails (we want this)
- The filename itself can be anything, and if it ends with any of the 3 above we strip it away

I believe this is correct